docs/metrics.md

```diff
@@ -5,42 +5,29 @@ SPDX-License-Identifier: Apache-2.0
 
 # Metrics
 
-Metrics are collected using Prometheus, and registered with its global metrics
-registry in each controller. The endpoint exposed is `localhost:9289/metrics`.
-If running from minikube or a container, expose the port using `port-forward`
-on the hub:
-
-```bash
-kubectl port-forward -n ramen-system \
-  service/ramen-hub-operator-metrics-service 9289:9289
-```
+Metrics are collected using Prometheus, and registered with its global
+metrics registry in each controller. There are two ways where you can
+look at the metrics in ramen. One way is use prometheus stack(recommended)
+and the other way is to use curl or postman. More details on
+each of these in the below sections.
 
-## Basic testing (no Prometheus required)
+More information on metrics is [here](https://book.kubebuilder.io/reference/metrics.html)
 
-Verify that the metrics endpoint is exposed with curl:
-`curl http://localhost:9289/metrics`
-
-If curl can connect, search for your metrics in the output.
+## 1. Using Prometheus Stack
 
-## Metrics List
-
-All metrics are prefixed with `ramen_`. This makes them easier to find.
+We recommend to use [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus).
+Installing this will give you a containerized stack that
+includes Prometheus, AlertManager and Grafana.
+Quickstart instructions are [here](https://github.com/prometheus-operator/kube-prometheus#quickstart).
 
-To get the list of all the Ramen metrics available and their descriptions,
-run the Ramen code, then run this command:
-`curl http://localhost:9289/metrics -s | grep "# HELP ramen_"`
+For more detailed information and querying,
+consider using the Prometheus Operator, but in summary:
 
-## Prometheus Stack
+### Setup
 
-For more detailed information and querying, consider using the Prometheus
-Operator
-[kube-prometheus](https://github.com/prometheus-operator/kube-prometheus).
-Installing this will give you a containerized stack that
-includes Prometheus, AlertManager and Grafana. Quickstart instructions are
-[here](https://github.com/prometheus-operator/kube-prometheus#quickstart),
-but in summary:
+Follow the next steps before configuring ramen.
 
-Setup and create:
+#### Installing kube-prometheus
```

Comment:
Point to docs from Prometheus rather than repeating the steps here.

Reply:
updated.